values.yaml

```diff
@@ -4,8 +4,19 @@ kobotoolbox:
   mongoName: "formhub"
   mongoDatabase: "mongodb://root:password@mongodb:27017"
   kobocatDatabase: "postgis://postgres:password@postgres-postgresql:5432/postgres"
-  redisSession: "redis://:pass@redis-master:6379/1"
-  serviceAccountBackend: "redis://:pass@redis-master:6379/1"
+  # Set to use for all redis needs (cache, broker, etc) redis connection string without db number.
+  # Unset to specify each redis usage seperately
+  # Redis default DB numbers
+  # - Enketo - 0 (cannot be changed)
+  # - Session - 1 
+  # - KPI Broker - 2
+  # - Kobocat Broker - 3
+  # - KPI Cache - 4
+  # - Kobocat Cache - 5
+  # - Service account - 6
```

Comment:
There are some difference with production servers: 

| DB | Helm | IOM | SCI | HHI (kobo-install) |
| ------------- | ------------- | ------------- | ------------- | ------------- |
| Session | 1 | ✅ | ✅ | 2 |
| KPI Broker | 2 | ✅ | ✅ | 1 |
| Kobocat Broker | 3 | ✅ | ✅ | 2 |
| KPI Cache | 4 | ✅ | 5 | 5 |
| Kobocat Cache | 5 | ✅ | 6 | 5 |
| Service account | 6 | 5 | ✅ | ✅ |

* CACHE_URL is shared with kobo-install